src/DotNet.Status.Web/DotNet.Status.Web.csproj

```diff
@@ -20,11 +20,16 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.AzureKeyVault" />
     <PackageReference Include="Octokit" />
     <PackageReference Include="WindowsAzure.Storage" /> 
+    <Compile Remove="tests\**" />
```

Comment:
Because of this, I think it's usually better to put the "test" project next to the project that's being tested. VS is going to fight tooth and nail to keep including the test project in the parent project, and it's easier not to fight with VS.

Reply:
Merged in the version of this project you created rather than the one I originally created :)